src/app/models/hris/EmployeeProfile/employeeProfileDetails.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { EmployeeAddress } from "../employee-address.interface";
+import { EmployeeData } from "../employee-data.interface";
+import { EmployeeSalary } from "../employee-salary.interface";
+import { EmployeeProfileContact } from "./employeeContactDetails.interface";
+import { EmployeePersonalDetails } from "./employeePersonalDetails.interface";
+
+export interface EmployeeProfileDetails {
+   employeeDetails : EmployeeProfileDetails,
+   personalDetails : EmployeePersonalDetails,
+   contactDetails : EmployeeProfileContact,
+   employeeData: EmployeeData,
+   photo: string;
+   active: boolean;
+   inactiveReason: string | null;
```

Comment:
Question: doesn't photo, active, inactiveReason exist in employeeProfileDetails?